packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -78,8 +79,9 @@ const getLabelFromField = (
     key: string | undefined,
 ) => {
     const item = findItem(fields, key);
+
     if (item) {
-        return getItemLabelWithoutTableName(item);
+        return getDateGroupLabel(item) || getItemLabelWithoutTableName(item);
```

Comment:
not sure where this exactly sits on the code, but see here: 
<img width="1463" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/4cc94d5c-0d9d-473c-bba2-4f6bceaaf92e">
`X-axis label` contains `Week` even though it's not displayed on the chart. Do we want to keep it like that? 
You can edit it ofc, and that works, just a question on the initial state

Example: https://lightdash-pr-8130.onrender.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/tables/events?create_saved_chart_version=%7B%22tableName%22%3A%22events%22%2C%22metricQuery%22%3A%7B%22dimensions%22%3A%5B%22events_date_week%22%5D%2C%22metrics%22%3A%5B%22events_in_eur%22%5D%2C%22filters%22%3A%7B%7D%2C%22sorts%22%3A%5B%7B%22fieldId%22%3A%22events_date_week%22%2C%22descending%22%3Atrue%7D%5D%2C%22limit%22%3A500%2C%22tableCalculations%22%3A%5B%5D%2C%22additionalMetrics%22%3A%5B%5D%7D%2C%22tableConfig%22%3A%7B%22columnOrder%22%3A%5B%22events_date_week%22%2C%22events_in_eur%22%5D%7D%2C%22chartConfig%22%3A%7B%22type%22%3A%22cartesian%22%2C%22config%22%3A%7B%22layout%22%3A%7B%22xField%22%3A%22events_date_week%22%2C%22yField%22%3A%5B%22events_in_eur%22%5D%7D%2C%22eChartsConfig%22%3A%7B%22series%22%3A%5B%7B%22type%22%3A%22bar%22%2C%22yAxisIndex%22%3A0%2C%22encode%22%3A%7B%22xRef%22%3A%7B%22field%22%3A%22events_date_week%22%7D%2C%22yRef%22%3A%7B%22field%22%3A%22events_in_eur%22%7D%7D%7D%5D%7D%7D%7D%7D

Reply:
Fixed 
![Screenshot from 2023-11-29 16-18-46](https://github.com/lightdash/lightdash/assets/1983672/1dd4cdd6-34ba-4200-b7a2-1f62fb29bc03)